packages/opentelemetry-core/src/platform/browser/BaseMetricPlugin.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { BaseAbstractMetricPlugin } from '../BaseAbstractMetricPlugin';
+import {
+  MetricPlugin,
+  MetricPluginConfig,
+  MeterProvider,
+} from '@opentelemetry/api';
+
+export abstract class BaseMetricPlugin<T> extends BaseAbstractMetricPlugin<T>
```

Comment:
Comments for this classes purpose. Especially note that patch needs to be implemented